app/serializers/shipit/stack_serializer.rb

```diff
@@ -5,7 +5,7 @@ class StackSerializer < ActiveModel::Serializer
     has_one :lock_author
     attributes :id, :repo_owner, :repo_name, :environment, :html_url, :url, :tasks_url, :deploy_url, :pull_requests_url,
                :deploy_spec, :undeployed_commits_count, :is_locked, :lock_reason, :continuous_deployment, :created_at,
-               :updated_at, :locked_since, :last_deployed_at, :branch, :merge_queue_enabled
+               :updated_at, :locked_since, :last_deployed_at, :branch, :merge_queue_enabled, :last_deploy_revision
```

Comment:
IMO it would be better to have `last_deployed_commit`, and let the commit serializer do the job so that you have `stack.last_deployed_commit.sha`